docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+#https://docs.docker.com/compose/compose-file/compose-versioning/
+version: "3.3"
+
+services:
+#  Fix startup of redis-stream at module load time and uncomment to have entire system in docker
+#  app:
+#    build: ./
+#    environment:
+#      - REDIS_HOST=redis
+#      - RHOST=redis
+#    networks:
+#      - redis-net
+#    depends_on:
+#      - redis
+#      - ratingsstream
+#    ports:
+#      - 9000:9000
+#      - 9443:9443
+#    command: /svc/bin/start -Dhttps.port=9443 -Dredis.ip="$REDIS_IP" -Dplay.crypto.secret=secret
```

Comment:
how do you start the server HTTPS from play framework ?

Reply:
I took out Play for now from docker-compose. We will start it separately. This is temporary.